### ISSUE TITLE
Publish snapshots

### ISSUE THREAD

**Issue by deepy** [node-gradle/gradle-node-plugin#62]
We should probably publish the snapshots somewhere like JCenter.

This is related to #15 and #10

**Comment by bsautel** [comment]
What is the use case for plugin snapshots?

With Gradle, it's easy to use the plugin source by adding:

```gradle
buildFile "/path/to/gradle-node-plugin"
```

**Comment by deepy** [comment]
Composite builds work great, provided that you've got the source on the same machine.

If I wanted to try something in our internal CI env I'd need a snapshot, if I wanted to share an early build with someone else.

**Comment by bsautel** [comment]
Yes, indeed, composite builds are not very convenient if you want to use them in CI.

The idea is to automatically publish to a snapshot repository if the CI build is passing, right?

**Comment by deepy** [comment]
Yeah, along with the other jars like javadoc and sources

**Comment by iherasymenko** [comment]
I'd suggest making the build compatible with jitpack.io
